deepchem/data/tests/test_fasta_loader.py

```diff
@@ -16,7 +16,9 @@ def setUp(self):
     super(TestFASTALoader, self).setUp()
     self.current_dir = os.path.dirname(os.path.abspath(__file__))
 
-  def test_fasta_load(self):
+  def test_fasta_one_hot(self):
+    legacy = False  # Whether to assume legacy one hot encoding shape from FASTA loader.
```

Comment:
If something is legacy, we should deprecate it explicitly. Have we changed the expected shapes?

Reply:
Yes, the shapes changed.